master/static/srv/get_trial_log_fields.sql

```diff
@@ -0,0 +1,7 @@
+select distinct
+                agent_id,
+                container_id,
+                rank_id,
+                std_type,
+                source
+from trial_logs where trial_id = $1;
```

Comment:
**question:** it's been a while since my last stay in SQLand, but won't this return every distinct permutation of all of the 5 fields? So let's assume we have `N` distinct values for each of the 5 field, won't this return `N^5` different rows?

Reply:
I should probably just try this out to see what gets returned...